test/e2e/networks/ci.toml

```diff
@@ -17,6 +17,7 @@ abci_protocol = "builtin"
 prometheus = true
 peer_gossip_intraloop_sleep_duration = "50ms"
 abci_tests_enabled = true
+constant_flip = true
```

Comment:
Should this be default? Just double checking.

Reply:
Yes IMO. We want expose this to all PRs (as it is more advanced as test, and doesn't mean the test will be longer).